tests/wicked/before_test.pm

```diff
@@ -47,7 +47,13 @@ sub run {
     if (check_var('IS_WICKED_REF', '1')) {
         record_info('INFO', 'Setup DHCP server');
         zypper_call('--quiet in dhcp-server', timeout => 200);
-        $self->get_from_data('wicked/dhcp/dhcpd.conf', '/etc/dhcpd.conf');
+        if (check_var('WICKED', '2nics')) {
+            record_info('Info', 'Set up 2nics DHCP and second ip');
+            assert_script_run(sprintf("ip a a %s dev %s", $self->get_ip(type => 'second_card'), $ctx->iface()));
```

Comment:
shouldn't this only be for ref?

Reply:
yes and we are inside `if(check_var('IS_WICKED_REF','1'))`